lib/validation.js

```diff
@@ -35,7 +36,7 @@ function build (opts) {
   }
 
   if (opts.schema.body) {
-    opts[bodySchema] = ajv.compile(opts.schema.body)
+    opts[bodySchema] = validatorMaker(opts.schema.body)
```

Comment:
can we just call `validatorMaker` `compile`?

Reply:
done